src/test/java/reposense/util/FileUtilTest.java

```diff
@@ -0,0 +1,34 @@
+package reposense.util;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class FileUtilTest {
+
+    private static final Path OUTPUT_DIRECTORY_ABSOLUTE = new File(FileUtilTest.class.getClassLoader()
+            .getResource("reposense-report-test").getFile()).toPath().toAbsolutePath();
+    private static final Path ARCHIVE_JSON_PATH = Paths.get(OUTPUT_DIRECTORY_ABSOLUTE.toString(), "archiveJSON.zip");
+
+    @Test
+    public void zipTest_success() {
+        FileUtil.zipJson(OUTPUT_DIRECTORY_ABSOLUTE);
+        Assert.assertTrue(Files.exists(ARCHIVE_JSON_PATH));
```

Comment:
Quick remark - 
Is this sufficient to know that your function is working well? What if the result does not meet your expectation?

Reply:
I thought about it too, but to test the content of the zip file we need to unzip it which makes it dependent on unzip(...) so I'm looking for an alternate method.

Reply:
alright, @AdityaA1998 please highlight when its ready for next iteration of review.